wagtail/models/__init__.py

```diff
@@ -2824,6 +2824,7 @@ def delete(self):
 
         return super().delete()
 
+    @transaction.atomic
```

Comment:
i guess if our caching is (to an extent) now reliant on forked wagtail functionality, we should definitely document this when we wrap up

Reply:
yeah, will make sure this is documented